service-front/module/Application/view/application/general/home/index.twig

```diff
@@ -11,6 +11,14 @@
 
     <p>We’ll save your LPA as you progress so you do not need to complete it in one go.</p>
 
+    <br>
+    <div class=panel>
```

Comment:
missing "s?

Reply:
Added now, my bad! However it works without the " too.